velox/exec/GroupingSet.cpp

```diff
@@ -948,15 +948,8 @@ void GroupingSet::spill() {
         makeSpillType(),
         rows->keyTypes().size(),
         std::vector<CompareFlags>(),
-        spillConfig_->getSpillDirPathCb,
-        spillConfig_->updateAndCheckSpillLimitCb,
-        spillConfig_->fileNamePrefix,
-        spillConfig_->writeBufferSize,
-        spillConfig_->compressionKind,
-        memory::spillMemoryPool(),
-        spillConfig_->executor,
-        spillConfig_->maxSpillRunRows,
-        spillConfig_->fileCreateConfig);
+        spillConfig_,
+        memory::spillMemoryPool());
```

Comment:
Can we also remove memory::spillMemoryPool() from the ctor as we always use spillMemoryPool? Thanks!